components/library/openssl/openssl-3.1/patches/042-bn_sqr_mont.patch

```diff
@@ -0,0 +1,13 @@
+See ../../common/patches/042-bn_sqr_mont.patch for details.
```

Comment:
I think we do not have such file.

Reply:
copied original comment from solaris-userland